validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java

```diff
@@ -236,6 +239,14 @@ public static Path getSlashingProtectionPath(final DataDirLayout dataDirLayout)
     return dataDirLayout.getValidatorDataDirectory().resolve("slashprotection");
   }
 
+  public static Path getKeystoreValidatorPath(final DataDirLayout dataDirLayout) {
+    return dataDirLayout.getValidatorDataDirectory().resolve("keystores");
+  }
+
+  public static Path getKeystorePasswordValidatorPath(final DataDirLayout dataDirLayout) {
+    return dataDirLayout.getValidatorDataDirectory().resolve("keystore-passwords");
+  }
```

Comment:
ok since i really do feel quite strongly about this...
 - these functions should make it clear that they are selecting our mutable keystore path / mutable keystore password path
 - I still believe the better paths would be something like `alterable-keystore`, `alterable-passwords`, or something to make it clear that its not just a general keystore. (or variable, erratic, mercurial but alterable or variable are probably better)